internal/proxy/options.go

```diff
@@ -125,6 +127,9 @@ func (o *Options) Validate() error {
 	if o.ProviderURLString == "" {
 		msgs = append(msgs, "missing setting: provider-url")
 	}
+	if o.ProxyProviderURLString == "" {
```

Comment:
not positive this is the best place to do this.  first time in this codebase though.

Reply:
this seems fine to me, can we add a test for this behavior?

Reply:
i stuck in a test for this